DESCRIPTION

```diff
@@ -17,6 +17,11 @@ Authors@R:
            role = "ctb",
            email = "kdavidson@bnl.gov",
            comment = c(ORCID = "0000-0001-5745-9689")),
+    person(given = "Kim",
+           family = "Ely",
+           role = "ctb",
+           email = "kely@bnl.gov",
+           comment = c(ORCID = "0000-0002-3915-001X")),
```

Comment:
@regnans just wanted you to check my updates to DESCRIPTION

Reply:
those details are correct (although I am not sure if that is what you were asking???)

Reply:
yeah that was it thanks!.  I am still working on preparing this example.